utils/env.go

```diff
@@ -12,8 +12,11 @@ type EnvProvider interface {
 }
 type envProvider struct{}
 
-var GetEnvProviderFunc func() EnvProvider
-var e EnvProvider
+//nolint:gochecknoglobals // need to test this in a future PR
```

Comment:
same here, link to a future work-item for tracking for improving test coverage on this and link with AB#12345 format.

Reply:
https://thycotic.visualstudio.com/Delinea.Work/_workitems/edit/561862